projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -88,21 +95,22 @@
   "item.pamhc2foodextended.grapefruitsodaitem": "西柚汽水",
   "item.pamhc2foodextended.grapesodaitem": "葡萄汽水",
   "item.pamhc2foodextended.hazelnutchocolateitem": "榛子巧克力酱",
-  "item.pamhc2foodextended.lemonlimesodaitem": "柠檬酸橙汽水",
+  "item.pamhc2foodextended.lemonlimesodaitem": "柠檬汽水",
   "item.pamhc2foodextended.maplecandiedbaconitem": "枫糖培根",
   "item.pamhc2foodextended.marzipanitem": "杏仁蛋白软糖",
   "item.pamhc2foodextended.orangesodaitem": "橘子汽水",
   "item.pamhc2foodextended.peanutbuttercupitem": "花生巧克力杯",
   "item.pamhc2foodextended.peppermintitem": "薄荷糖",
   "item.pamhc2foodextended.popcornitem": "爆米花",
-  "item.pamhc2foodextended.pralinesitem": "山核桃糖",
+  "item.pamhc2foodextended.pralinesitem": "碧根果糖",
   "item.pamhc2foodextended.rootbeerfloatitem": "雪顶根汁啤酒",
   "item.pamhc2foodextended.rootbeersodaitem": "根汁汽水",
   "item.pamhc2foodextended.sesamesnapsitem": "芝麻糖片",
   "item.pamhc2foodextended.sourcreamandonionpotatochipsitem": "酸奶油洋葱味薯片",
   "item.pamhc2foodextended.strawberrysodaitem": "草莓汽水",
   "item.pamhc2foodextended.theatreboxitem": "影院套餐",
   "item.pamhc2foodextended.ediblerootitem": "食用块茎",
+
   "item.pamhc2foodextended.baconmushroomburgeritem": "培根蘑菇汉堡",
   "item.pamhc2foodextended.bamboosteamedriceitem": "竹笋蒸饭",

```

```suggestion
  "item.pamhc2foodextended.bamboosteamedriceitem": "竹蒸饭",
```
按理说竹笋应该是`bamboo shoot`才对...个人倒更倾向于竹筒饭，但贴图完全不像是...